packages/zoe/src/contracts/priceAggregatorChainlink.js

```diff
@@ -465,20 +473,28 @@ const start = async (
       canSupersede = supersedable(subtract(_roundId, 1), blockTimestamp);
     }
 
-    if (startingRound === 0n) return 'not enabled oracle';
-    if (startingRound > _roundId) return 'not yet enabled oracle';
-    if (oracleStatuses.get(_oracle).endingRound < _roundId)
+    if (startingRound === 0n) {
+      return 'not enabled oracle';
+    }
+    if (startingRound > _roundId) {
+      return 'not yet enabled oracle';
+    }
+    if (oracleStatuses.get(_oracle).endingRound < _roundId) {
       return 'no longer allowed oracle';
```

Comment:
This is an example of the case that scares me: where the arm is on its own line and indented.